dwave/cloud/testing/mocks.py

```diff
@@ -357,7 +352,34 @@ def hybrid_bqm_solver_data(**kwargs) -> dict:
     params = dict(
         id="hybrid_bqm_solver",
         description="Hybrid unstructured BQM mock solver",
-        supported_problem_types=["bqm"]
+        supported_problem_types=["bqm"],
+        # solver-specific properties (mock values)
+        maximum_number_of_biases=2**20,
+        maximum_number_of_variables=2**16,
+        maximum_time_limit_hrs=1.0,
+        minimum_time_limit=[[1, 1.0], [1024, 2.0], [4096, 4.0], [8192, 16.0]],
+    )
+    params.update(**kwargs)
+    return unstructured_solver_data(**params)
+
+
+def hybrid_cqm_solver_data(**kwargs) -> dict:
+    params = dict(
+        id="hybrid_cqm_solver",
+        description="Hybrid unstructured CQM mock solver",
+        supported_problem_types=["cqm"],
+        # solver-specific properties (mock values)
+        maximum_number_of_biases=2**20,
```

Comment:
Why not 2e07 etc for these?

Reply:
Just adding dummy values (for now). I just wanted to capture the list of properties and order of magnitude for values..